tests/integration/test_format_txts_e2e.py

```
@@ -0,0 +1,266 @@
+    
+    phone_file.write_text(phone_content)
+    email_file.write_text(email_content)
+    
+    # Return the paths and a cleanup function
+    yield {
+        "export_dir": export_dir,
+        "output_dir": output_dir,
+        "contacts_file": contacts_file,
+        "phone_file": phone_file,
+        "email_file": email_file,
+    }
```

Ruff (0.14.2)

No teardown in fixture `setup_test_environment`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)